dotcom-rendering/src/lib/decideContainerOverrides.ts

```diff
@@ -43,21 +43,23 @@ const textCardHeadline = (
 const textCardStandfirst = textCardHeadline;
 const textCardFooter = textCardHeadline;
 
-const textContainerFooter = (
-	containerPalette: Extract<DCRContainerPalette, 'Branded'>,
-): string => {
+const textContainerFooter = (containerPalette: DCRContainerPalette): string => {
 	switch (containerPalette) {
 		case 'Branded':
 			return neutral[46];
```

Comment:
Can we just remove the case for Branded now?

Reply:
I guess the idea here was that we have made an explicit decision for 'Branded' but not for the other palettes, let's discuss in this afternoon session if you can join at 2? - happy to remove tho it's probably pre-empting too much?

Reply:
we think this will be addressed as part of the refactor mentioned by Max above ^